_raml/apps/tax-model/examples/get_taxmodel_response.json

```diff
@@ -1,5 +1,7 @@
 {
   "model": "gross",
   "displayTaxes": true,
-  "noTaxesMessage":"According to § 19 UStG we do not charge the German sales tax, and consequently do not levy it (regulation for small business entrepreneurs)."
+  "noTaxesMessage": "According to § 19 UStG we do not charge the German sales tax, and consequently do not levy it (regulation for small business entrepreneurs).",
+  "deliveryThresholdIsActive": false,
```

Comment:
I've removed the quotes here as the "true" above also has none. Please tell me if that was wrong @mjger :)